README.md

```diff
@@ -361,5 +344,29 @@ You will also find the following intermediate files:
 
 ## Debugging
 
-To debug some docker logs, set `DEBUG_DOCKER=true`.
-To debug the localstack docker container startup, set `DEBUG_LOCALSTACK=true`.
+To enable debug level logging we are using the [debug][debug] library to create the log lines.  
+There is also an environment variable `ENABLE_LAMBDA_LOGGING` available to
+print the results of a lambda run, both `stdout` and `stderr`
+Available flags are 
+- `lambda-tools:lambda`
+- `lambda-tools:docker`
```

Comment:
I think there are `localstack` namespaced debug level logs as well.

Reply:
Adding that.  Thanks for the catch.